src/pudl_archiver/archivers/ferc/xbrl.py

```diff
@@ -35,7 +35,7 @@
 to this URL to specify the month and year desired.
 """
 
-Year = pydantic.conint(ge=2000, le=datetime.datetime.today().year)
+Year = Annotated[int, Field(ge=2000, le=datetime.datetime.today().year)]
```

Comment:
I think it's strictly possible to get years pre-2000 (if these are report years, not publication years), since companies sometimes update data from many years ago, and all of those revisions will be published as XBRL now, and the data goes back further than 2000.  FERC Form 1 in digital form goes as far as 1994.

Reply:
Updated to allow 1994 and on.